<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alexander Bürger</author>
 <class>WebMapDialog</class>
 <widget class="QWidget" name="WebMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web Maps</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabsTop">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAdd">
      <attribute name="title">
       <string>Add</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackAdd">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="addPageService">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="labelAddChooseService">
             <property name="text">
              <string>Choose service:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editAddServiceFilter">
             <property name="placeholderText">
              <string>Type to filter service name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listAddServices">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="addPageLayer">
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,1">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="labelAddChooseLayer">
             <property name="text">
              <string notr="true" extracomment="translated in source code">Choose layer in service %1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editAddLayerFilter">
             <property name="placeholderText">
              <string>Type to filter layer name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPending">
             <property name="text">
              <string>Waiting for webmap service …</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listAddLayers">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonRestart">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Start over (from service selection)</string>
           </property>
           <property name="text">
            <string>Restart</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBack">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Back to the previous step</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNextAdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string notr="true">Next/Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStyle">
      <attribute name="title">
       <string>Modify</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboStyleLayer">
           <property name="toolTip">
            <string>Choose the layer to modify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonLayerRemove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Remove</string>
           </property>
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Style</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutStyle" columnstretch="0,1,1,1">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelStyle">
            <property name="text">
             <string>Style name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="comboStyle">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Choose a style provided by the webmap service</string>
            </property>
           </widget>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="labelGrey">
            <property name="text">
             <string>Make grey:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkGrey">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Transform webmap to greyscale</string>
            </property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QLabel" name="labelAlpha">
            <property name="text">
             <string>Change transparency:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinAlphaScale">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Scale layer transparency with this value</string>
            </property>
            <property name="prefix">
             <string notr="true">*</string>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="spinAlphaOffset">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Add this to layer transparency after scaling</string>
            </property>
            <property name="prefix">
             <string notr="true">+</string>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacerAlpha">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>

          <item row="3" column="0">
           <widget class="QLabel" name="labelPlotOrder">
            <property name="text">
             <string>Drawing order:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3">
           <widget class="QComboBox" name="comboPlotOrder">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Choose Z order for this layer.</string>
            </property>
            <item>
             <property name="text">
              <string>Background</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shade background</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shade</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Lines Background</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Lines</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <widget class="QGroupBox" name="groupTime">
         <property name="title">
          <string>Time</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelTimeOffset">
            <property name="text">
             <string>Offset:</string>
            </property>
            <property name="buddy">
             <cstring>spinTimeOffset</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinTimeOffset">
            <property name="toolTip">
             <string>Time offset relative to plot time</string>
            </property>
            <property name="suffix">
             <string notr="true">s</string>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="labelTimeTolerance">
            <property name="text">
             <string>Tolerance:</string>
            </property>
            <property name="buddy">
             <cstring>spinTimeTolerance</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinTimeTolerance">
            <property name="toolTip">
             <string>Time matching tolerance</string>
            </property>
            <property name="specialValueText">
             <string>nearest</string>
            </property>
            <property name="suffix">
             <string notr="true">s</string>
            </property>
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QCheckBox" name="checkTimeFixed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Check to fix time to a specific value</string>
            </property>
            <property name="text">
             <string>Fixed:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboTimeFixed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Fixed time value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupStyleOtherDims">
         <property name="title">
          <string>Other dimensions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableOtherDims">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushModifyReset">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonModifyApply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>groupTime</zorder>
      <zorder>groupStyle</zorder>
      <zorder>groupStyleOtherDims</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <tabstops>
  <tabstop>tabsTop</tabstop>
  <tabstop>editAddServiceFilter</tabstop>
  <tabstop>listAddServices</tabstop>
  <tabstop>buttonNextAdd</tabstop>
  <tabstop>buttonBack</tabstop>
  <tabstop>buttonRestart</tabstop>
  <tabstop>comboStyleLayer</tabstop>
  <tabstop>spinAlphaScale</tabstop>
  <tabstop>spinAlphaOffset</tabstop>
  <tabstop>spinTimeOffset</tabstop>
  <tabstop>spinTimeTolerance</tabstop>
  <tabstop>checkTimeFixed</tabstop>
  <tabstop>comboTimeFixed</tabstop>
  <tabstop>listAddLayers</tabstop>
  <tabstop>editAddLayerFilter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkGrey</sender>
   <signal>toggled(bool)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyGreyToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboStyleLayer</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyLayerSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonModifyApply</sender>
   <signal>clicked()</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushModifyReset</sender>
   <signal>clicked()</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAlphaScale</sender>
   <signal>valueChanged(double)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyAlphaScaleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAlphaOffset</sender>
   <signal>valueChanged(double)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyAlphaOffsetChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboPlotOrder</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyPlotOrderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonLayerRemove</sender>
   <signal>clicked()</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyLayerRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinTimeOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyTimeOffsetChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinTimeTolerance</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyTimeToleranceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboStyle</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>WebMapDialog</receiver>
   <slot>onModifyStyleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onModifyGreyToggled()</slot>
  <slot>onModifyLayerSelected()</slot>
  <slot>onModifyApply()</slot>
  <slot>onModifyReset()</slot>
  <slot>onModifyAlphaScaleChanged()</slot>
  <slot>onModifyAlphaOffsetChanged()</slot>
  <slot>onModifyPlotOrderChanged()</slot>
  <slot>onModifyLayerRemove()</slot>
  <slot>onModifyTimeToleranceChanged()</slot>
  <slot>onModifyTimeOffsetChanged()</slot>
  <slot>onModifyStyleChanged()</slot>
 </slots>
</ui>
